### DISCUSSION TITLE
How do you handle passwords?

### DISCUSSION THREAD

**Discussion by velllu** [pocketbase/pocketbase#2967] [Q&A] [answered]
With the default users collection, there is a username & an email, I don't see any password field

**Comment by ganigeorgiev** [comment, marked as answer]
The password is stored as bcrypt hash in the internal `passwordHash` field (it is not exposed over the web apis because users usually don't interact directly with it).

Currently the bcrypt weight/cost factor is hardcoded as `12` but eventually in the future we may provide an auth collection option to allow users to adjust it to accommodate better to the server resources.
